### ISSUE TITLE
No way to allow use of items

### ISSUE THREAD

**Issue by aqua-air** [thexaero/open-parties-and-claims#361]
When using this on servers there are hundereds of items we need players to be able to interact with in every claim. This mod automatically blacklists everything including ender chests, grave stones, etc.. 
Sometimes even putting a block on the forced allow list does not allow it's use. Then we have to restart the server every time we want to edit the list of allowed items. The config for the mod is in utmost need of reorganizing and clarifying.  Adding graves, ender chests, shops, half the time adding it to the allow list does not even make it work. We need an in-game command to manage this list because this is extremely time consuming and annoying to restart the server to edit a config that already unclear

TLDR = Suggesting in game config options and the config explanations to be updated with clarity

**Comment by thexaero** [comment]
There is a way. And stop blaming me for your mistakes. Thanks!
